Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,3 +42,45 @@
     from FastSimulation.Configuration.DigiAliases_cff import loadDigiAliases
     loadDigiAliases(premix = False)
     from FastSimulation.Configuration.DigiAliases_cff import generalTracks,ecalPreshowerDigis,ecalDigis,hcalDigis,muonDTDigis,muonCSCDigis,muonRPCDigis
+
+#phase 2 common mods
+def _modifyDigitizerPhase2Common( theProcess ):
+    theProcess.load("CalibCalorimetry/HcalPlugins/Hcal_Conditions_forGlobalTag_cff")
+    theProcess.es_hardcode.toGet = cms.untracked.vstring(
```

Comment:
So for now we don't have a special mechanism in the eras to handle addition or removal of ES modules (it can be done on event modules via addition or subtraction from paths). So for `es_hardcode` I'd say keep it in a makeProcessModifier but move the manipulation of `simHcalDigis` and `hcalDigiSequence` to where those items are defined and manipulate the objects instead of using a process modifier. I'll add an issue about how to handle new ES modules with eras.